src/App.tsx

```diff
@@ -43,18 +43,23 @@ const App = () => {
   const TOKEN = localStorage.getItem(ACCESS_TOKEN);
   const isAuthorized = useRecoilValue($isAuthorized) || !!TOKEN;
   const toast = useRecoilValue($toast);
+  const generationNumber = useRecoilValue($generationNumber);
+
+  const getTeams = useRecoilCallback(({ set }) => async () => {
+    const { data: teams } = await api.getTeams(generationNumber);
+
+    set($teams, teams);
+  });
```

Comment:
useEffect 없애고 이렇게 하면 되지 않을까요?
```suggestion
  useRecoilCallback(
    ({ snapshot, set }) =>
      async () => {
        const generationNumberSnapshot = snapshot.getLoadable($generationNumber).contents;
        const { data: teams } = await api.getTeams(generationNumberSnapshot);
        set($teams, teams);
      },
    [],
  )();
```

Reply:
요 생각도 했었는데 요렇게 처리했을 때 렌더링 될때 마다 api.getTeams 가 호출되는 문제가 있어요 ㅠ

Reply:
일단 현상태 유지하기로 했습니당